src/data/CHDatasource.ts

```diff
@@ -791,3 +925,22 @@ interface Tags {
   type?: TagType;
   frame: DataFrame;
 }
+
+export interface LogContextColumn {
+  name: string;
+  value: string;
+}
+
+/*
+ * Can't import these types from anywhere from any version, but they are required for log context:
+ * from @grafana/data https://github.com/grafana/grafana/blob/4f04b5849e72acd33fb6d621dacf860abf1305e8/packages/grafana-data/src/types/logs.ts#L132-L140
+ */
+interface LogRowContextOptions {
```

Comment:
This type and the type below should be importable if we upgrade `@grafana/data` to 10+. We should then update the rest of the packages also however this may lead to some API changes.

Reply:
If we upgrade to 10.x on these modules, do we need to also upgrade our minimum required Grafana version from 9.4.3?

Reply:
We shouldn't have to, atm I don't believe we're using any Grafana 10 specific functionality in this plugin

Reply:
I updated the modules and fixed the imports

Reply:
Nice! There's one type that needs to be fixed and then the build should pass and we can merge this!